metadata_annotation_farland.ipynb

```diff
@@ -0,0 +1,2307 @@
+{
```

Comment:
<p>What's the difference between <code>bulk_rna</code> and <code>ccle_expression</code> since they're both RNA?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>